tests/lib/managers/files-test.js

```diff
@@ -698,6 +749,41 @@ describe('Files', function() {
 		});
 	});
 
+	describe('uploadNewFileVersionFromPath()', function() {
+
+		var	PATH = './folders-test.js';
+
+		it('should call BoxClient.upload() with the correct non-callback params', function() {
+			sandbox.stub(boxClientFake, 'defaultResponseHandler');
+			sandbox.mock(boxClientFake).expects('upload').withArgs('/files/1234/content', null);
```

Comment:
This is not the correct arguments that we expect to be passed to this mock!  It should not get null for the file content!